tenbilac/com.py

```diff
@@ -89,9 +90,42 @@ def __init__(self, configpath, configlist=None):
 		# For easy access, we point to a few configuration items:
 		self.inputnormerpath = os.path.join(self.workdir, "input_normer.pkl")
 		self.targetnormerpath = os.path.join(self.workdir, "target_normer.pkl")
-	
+			
 		logger.info("Constructed Tenbilac '{self.name}' with workdir '{self.workdir}'.".format(self=self))
+		
+		# Now we take care of the logging, in a brutal way. The methods will redicrec all tenbilac log to a file.
+		# The code below only sets the logging up. It has to be activated and deactivated inside the methods.
+		# Did not yet find the optimal way of doing this (decorator as class method ?)
+		if self.config.getboolean("setup", "logtofile"):
+			self.logger = logging.getLogger("tenbilac")
+			logpath = os.path.join(self.workdir, "log.txt")
```

Comment:
Good ideas -- I agree, but would say let's leave this to when we fix/improve the logging. Currently we only log the training to file anyway. I guess this is not a priority for this paper (but of course, if we see elegant solutions, we could implement them)